package.json

```diff
@@ -38,6 +38,8 @@
     "ioredis": "^4.17.3",
     "lodash": "^4.17.15",
     "logform": "^2.2.0",
+    "moment": "^2.29.1",
+    "moment-timezone": "^0.5.32",
```

Comment:
Don't at me with this one, don't care how big it is, I'd rather have a vendor base with this bloat in than FIGURING OUT TIMEZONES WITH JAVASCRIPT NATIVE DATES.

Also yes I know there's other lighter libraries out of there, but in the interest of time and effort I'm sticking with this. Feel free to propose a PR with a new library in.